forena.report.inc

```diff
@@ -1420,9 +1420,9 @@ function forena_report_add_parameter_form($form, &$form_state, $report_name) {
   );
 
   $form['cancel'] = array(
-      '#type' => 'submit',
-      '#value' => t('Cancel'),
-      '#submit' => array('forena_update_cancel'),
+       '#type' => 'link',
+       '#href' => "reports/$report_name/edit",
+       '#title' => 'Cancel'
```

Comment:
Be sure to wrap constant strings in the `t()` function to make them translatable.

Reply:
Also, watch change in indentation - from 4 characters to 5.